docs/Comparison.md

```diff
@@ -2,23 +2,24 @@
-|Regular updates delay                       | About 1 business day     | About 1 business day                           | About 1 business day                        | Upstream of RHEL  | About 1 business day |
-|Last minor version release delay            | [5 days][alma9.4]         | [6 days][oracle9.4]                            | [8 days][rocky9.4]                          | N/A               | N/A               |
-|Errata                                      | Yes                      | Yes                                            | Yes                                         | No                | No                |
-|Lifecycle                                   | 10 Years                 | 10 Years                                       | 10 Years                                    | 5-6 Years         | EOL on 2021-12-31 |
-|Commercial support                          | 3rd party                | Oracle, 3rd parties                            | 3rd Party                                   | 3rd party         | 3rd party         |
-|Livepatching service                        | KernelCare               | Oracle Ksplice, KernelCare Available           | KernelCare Available                        | Not available     | KernelCare, Kpatch |
-|FIPS compliance                             | Yes                      | Yes                                            | Planned                                     | Not available     | Not available     |
-|ARM support                                 | Yes                      | Yes                                            | Yes                                         | Yes               | Yes               |
-|IBM PowerPC support                         | Yes                      | No                                             | Yes (versions ≥ 9)                          | Yes               | Yes               |
-|IBM Z (s390x) support                       | Yes                      | No                                             | Yes (versions ≥ 9)                          | No                | No                |
-|SecureBoot                                  | Yes                      | Yes                                            | Yes                                         | Yes               | Yes               |
-|Owned By:                                   | AlmaLinux OS Foundation  | Oracle Inc                                     | Rocky Enterprise Software Foundation        | Red Hat Inc       | Red Hat Inc       |
-|Owned by org type:                          | Non-Profit 501(c)6       | For Profit C-Corp                              | For Profit, Public Benefit Corp             | For Profit C-Corp | For Profit C-Corp |
+> Diagram by [Carl George](https://www.linkedin.com/in/carlwgeorge/)
+
+![relationship_chart](/images/Comparison-relationship_chart.png)
```

Comment:
I'd like to see this image updated to be AlmaLinux-specific. If that's outside your wheelhouse, let me know! I can create one for you.

Reply:
It would be great if you could create one, I tried creating something prior to this PR but it was horrible :laughing:.